flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BucketWriter.java

```diff
@@ -459,6 +464,7 @@ public Void call() throws Exception {
    * @throws java.io.IOException
    * @throws InterruptedException
    */
+  @SuppressWarnings({"squid:S1860","squid:MethodCyclomaticComplexity","squid:S134"})
   public synchronized void append(final Event event)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.io.IOException, java.lang.InterruptedException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1160)